{"version":3,"file":"CancelledError-FjP5S_cR.js","names":["message: string"],"sources":["../../src/common/api/common/error/CancelledError.ts"],"sourcesContent":["//@bundleInto:common-min\n\nimport { TutanotaError } from \"@tutao/tutanota-error\"\n\nexport class CancelledError extends TutanotaError {\n\t/**\n\t * A cancelled error is thrown when a async action is aborted\n\t * @param message An information about the exception.\n\t * @param error The original error that was thrown.\n\t */\n\tconstructor(message: string) {\n\t\tsuper(\"CancelledError\", message)\n\t}\n}\n"],"mappings":";;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;IAIa,iBAAN,cAA6B,cAAc;;;;;;CAMjD,YAAYA,SAAiB;AAC5B,QAAM,kBAAkB,QAAQ;CAChC;AACD"}